.circleci/run_docker_mingw.sh

```diff
@@ -15,7 +15,7 @@ CXXFLAGS="-D_hypot=hypot -D_GLIBCXX_ASSERTIONS" ${ARCH}-w64-mingw32-cmake \
   -DPYTHON_LIBRARY=${MINGW_PREFIX}/lib/libpython${PYMAJMIN}.dll.a \
   -DPYTHON_EXECUTABLE=/usr/bin/${ARCH}-w64-mingw32-python${PYMAJMIN}-bin \
   -DPYTHON_SITE_PACKAGES=Lib/site-packages \
-  -DUSE_TBB=OFF \
+  -DUSE_TBB=OFF -DCMAKE_CXX_FLAGS="-march=native" \
```

Comment:
I 'd rather not to have to use "march=native". What happens is that the ci uses mingw in it's 32 bits variant (64 is available too though). It works because the host processor actually is x86_64, so it has sse2. But the client machine could run an i686 processor and this will crash with an illegal instruction. 

A better way to do that is to explicitely test if sse2 is available:
```
try_compile (OPENTURNS_HAVE_SSE2
  ${CMAKE_CURRENT_BINARY_DIR}
  ${CMAKE_CURRENT_SOURCE_DIR}/cmake/sse2.cxx)
```

with cmake/sse2.cxx:
```
#include <emmintrin.h>

int main()
{
  __m128d v = _mm_setzero_pd();
  return 0;
}
```

Then we only use fmath only if OPENTURNS_HAVE_SSE2.

Reply:
ok